tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I've decided to use TypeScript here because it's very helpful for working in a team.